test/fixtures/rules/std_rule_cases/L054.yml

```diff
@@ -0,0 +1,301 @@
+rule: L054
```

Comment:
By `consistent` I was more thinking this should be caught:

```sql
    SELECT
        foo,
        bar,
        sum(baz) AS sum_value
    FROM fake_table
    ORDER BY
        foo, bar
    GROUP BY
        1, 2;
```

Is that caught?

Should we add a test case for this? And vice versa (positional ORDER BY, named GROUP BY)?

Reply:
good spot, I'll look at using memory to remember the other clauses.

Reply:
Added this in and working well :)